src/options.cpp

```diff
@@ -1012,6 +1012,11 @@ Options::Options()
     m_mmOutput.Init(false);
     this->Register(&m_mmOutput, "mmOutput", &m_general);
 
+    m_moveScoreDefinitionToStaff.SetInfo("Move score definition to staff",
+        "Move score definition (clef, keySig, meterSig, etc.) from scoreDef to staffDef");
+    m_moveScoreDefinitionToStaff.Init(false);
+    this->Register(&m_moveScoreDefinitionToStaff, "moveScoreDefinitionToStaff", &m_general);
+
```

Comment:
Why this without using it?

Reply:
What do you mean? The option is false by default but used with `--move-score-definition-to-staff`

Reply:
Never mind. I missed the spot in iomei.